types/ace-modules.d.ts

```diff
@@ -3948,6 +3953,11 @@ declare module "ace-code/src/edit_session" {
          * @param {String} className The class to add
          **/
         addGutterDecoration(row: number, className: string): void;
+        /**
+         * Hides the fold widget/icon from a specific row in the gutter
```

Comment:
We're adding an API to hide gutter icons on a row, but how can users unhide those hidden icons?